core/src/main/scala-3/chisel3/VerificationStatementIntf.scala

```diff
@@ -2,14 +2,65 @@
 
 package chisel3
 
-private[chisel3] trait Assert$Intf { self: assert.type =>
-  // TODO implement
+import chisel3.experimental.{BaseModule, SourceInfo}
+
+trait VerifPrintMacrosDoc
+
+val emptySourceLine: VerifStmtMacrosCompat.SourceLineInfo = ("???", 0)
```

Comment:
I talked with @adkian-sifive offline about this already.

I'm okay with this as an expedient path. However, please do three things:

1. Open a tracking issue on this repo about adding the missing macro to fix this.
2. Change the "???" to reference the issue number of (1).
3. Open an internal tracking ticket / include this in the internal tracking of the Scala 3 migration.

Ideally, there is _some_ test that will fail with this being incorrect and that would be good as it prevents us releasing a Scala 3 version without fixing this.